config/owasp/suppressions.xml

```diff
@@ -42,8 +42,8 @@
   </suppress>
 
   <suppress>
-    <notes>This is transitive dependency coming for spring boot, CVE is for tomcat-embed-core-9.0.31.jar and tomcat-embed-websocket-9.0.31.jar</notes>
-    <cve>CVE-2020-13943</cve>
+    <notes>This is transitive dependency coming for spring boot starter, CVE is for tomcat-embed-core-9.0.39.jar and tomcat-embed-websocket-9.0.39.jar</notes>
+    <cve>CVE-2020-17527</cve>
```

Comment:
upgrade tomcat then like everyone else =/

Reply:
@timja This is transitive dependency coming from spring boot starter, CVE is for tomcat-embed-core/Websocket jar. The latest version also has the same CVE issues. Don't have any other option apart from suppression.

Reply:
this doesn't seem to be correct, if you read the advisory you'll see only up to 39 has that issue
https://nvd.nist.gov/vuln/detail/CVE-2020-17527

41 is patched, if you update to that you will pick up the fix,
https://mvnrepository.com/artifact/org.apache.tomcat.embed/tomcat-embed-websocket/9.0.41

Reply:
@timja I have tried with the 10.0.0 version which had CVE present, but Yeah with 9.0.41 all sorted now. I have removed the suppression and upgraded tomcat versions.